### ISSUE TITLE
Remote images which return 403 when no user-agent is set

### ISSUE THREAD

**Issue by chrismytton** [jpmckinney/image-proxy#11]
I've run into an issue with an image which only returns 200 when a user-agent is set.

http://www.riigikogu.ee/?op=emsplain&page=pilt&pid=fe748f4d-3f50-4af8-8069-92a460978d2b

```
$ curl -sv -H 'User-Agent:' -o /dev/null 'http://www.riigikogu.ee/?op=emsplain&page=pilt&pid=fe748f4d-3f50-4af8-8069-92a460978d2b'
* Hostname was NOT found in DNS cache
*   Trying 195.80.106.68...
* Connected to www.riigikogu.ee (195.80.106.68) port 80 (#0)
> GET /?op=emsplain&page=pilt&pid=fe748f4d-3f50-4af8-8069-92a460978d2b HTTP/1.1
> Host: www.riigikogu.ee
> Accept: */*
>
< HTTP/1.1 403 Forbidden
< Date: Wed, 11 Mar 2015 12:46:19 GMT
* Server Apache is not blacklisted
< Server: Apache
< Vary: Accept-Encoding
< Content-Length: 45
< Content-Type: text/html; charset=iso-8859-1
<
{ [data not shown]
* Connection #0 to host www.riigikogu.ee left intact
```

So it seems that image-proxy needs to set the `User-Agent` header in order for this request to work correctly.

**Comment by jpmckinney** [comment]
Fixed in 0.0.4. Thanks for reporting!

**Comment by chrismytton** [comment]
Woo, thanks for the quick fix! :sparkles:
